packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
+    }
 
     return React.createElement(component, {
       ...fixture,
```

Comment:
> While writing the test I realized that children in React-Cosmos is not the same as children in React.

Not really. `fixture.children` are React Elements, not object with props. Not a very used pattern at HS, but useful when you create fixtures for wrapper components like modal windows.

I think you're thinking of the [loadChild](https://github.com/react-cosmos/react-cosmos/blob/b3be0680336724bce8830dae2f367df260394364/packages/react-component-tree/src/load-child.js) mixin where you define elements at the beginning of the component and then insert them in the render function using `loadChild`. That's just an optional utility, not used in any React Cosmos example.